iceoryx_binding_c/test/moduletests/test_log.cpp

```diff
@@ -30,28 +30,28 @@ using namespace iox::log;
 TEST(iox_log_test, LogLevelIsSetCorrectly)
 {
     ::testing::Test::RecordProperty("TEST_ID", "cbfc1b9d-d770-441c-b53d-11b1685ca0b4");
-    auto& logManager = iox::log::LogManager::GetLogManager();
+    auto& logger = iox::log::Logger::get();
 
     iox_set_loglevel(Iceoryx_LogLevel_Off);
-    EXPECT_EQ(logManager.DefaultLogLevel(), LogLevel::kOff);
+    EXPECT_EQ(logger.getLogLevel(), LogLevel::OFF);
```

Comment:
Personally I'm also not a fan of the `get` prefix but for whatever reason I added it here. Probably because @elfenpiff always asked me to do it and I wanted to save us some time in this PR :smile:

Reply:
No change required. Related: I like the properties approach from C# https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/auto-implemented-properties, which allows auto implemented or derived properties (which are also only functions in the end).

Reply:
yeah, C++ is not great in this regard